nucliadb/src/nucliadb/common/external_index_providers/base.py

```diff
@@ -59,16 +61,8 @@ class TextBlockMatch(BaseModel):
     needed in order to later hydrate retrieval results.
     """
 
-    id: str
-    resource_id: str
-    field_id: str
-    subfield_id: Optional[str] = None
-    index: Optional[int] = None
-    position_start: int
-    position_end: int
-    position_start_seconds: list[int] = []
-    position_end_seconds: list[int] = []
-    page_number: Optional[int] = None
+    id: ParagraphId
```

Comment:
Maybe rename this as `paragraph_id` or just `paragraph`? As it is now, looks like the id of the match, whatever that may mean. It confused me a little.